distribution_circuits_sale/models/time_frame.py

```diff
@@ -66,11 +73,19 @@ def action_cancel(self):
     def action_open(self):
         self.ensure_one()
         self.write({'state': 'open'})
+        if self.env['ir.config_parameter'].sudo().get_param(
+                    'distribution_circuits_website_sale.send_mail_to_supervisor'):
+            email_template_timeframe_success_state = self.env.ref('distribution_circuits_sale.email_template_timeframe_success_state', False)
+            email_template_timeframe_success_state.send_mail(self.id)
```

Comment:
did you test it being logged with the supervisor as a normal user and not the administrator ? cause in that case it could not receive the email. a solution could be to use sudo() to force the mail being sent by the admin

Reply:
The demo user does not have access to EMH. Sending using the administrator is fine actually, and since I have to go forward with this I implemented your suggestion of `sudo()` for safety, and added an extra test on the mail.